app/controllers/api/v1/profiles_controller.rb

```diff
@@ -15,19 +13,26 @@ def update
     @user.update!(profile_params)
   end
 
+  def availability
```

Comment:
new routes in the profile for the agent to update their availability. 
new route because now updating availability will requires the specific account id